edx_when/migrations/0005_auto_20190911_1056.py

```diff
@@ -1,7 +1,36 @@
 # Generated by Django 1.11.20 on 2019-09-11 10:56
 
+from django.db import connection, migrations, models
 
-from django.db import migrations, models
+
+def alter_rel_date_column(apps, schema_editor):
+    db_engine = connection.settings_dict['ENGINE']
+    with connection.cursor() as cursor:
+        if 'postgres' in db_engine:
+            cursor.execute(
+                'ALTER TABLE edx_when_datepolicy '
+                'ALTER COLUMN rel_date TYPE interval USING rel_date::interval;'
+            )
+        elif 'mysql' in db_engine:
+            cursor.execute(
+                'ALTER TABLE edx_when_datepolicy '
+                'MODIFY rel_date BIGINT;'
+            )
+
+
+def reverse_alter_rel_date_column(apps, schema_editor):
+    db_engine = connection.settings_dict['ENGINE']
+    with connection.cursor() as cursor:
+        if 'postgres' in db_engine:
+            cursor.execute(
+                'ALTER TABLE edx_when_datepolicy '
+                'ALTER COLUMN rel_date TYPE varchar USING rel_date::varchar;'
+            )
+        elif 'mysql' in db_engine:
+            cursor.execute(
+                'ALTER TABLE edx_when_datepolicy '
+                'MODIFY rel_date VARCHAR(255);'
+            )
```

Comment:
Can you please document in docstrings why these two functions are necessary at all? Won't Django's standard handling of DurationField create a bigint or interval as necessary?